pkg/docker/util.go

```diff
@@ -83,7 +83,7 @@ func StreamContainerIO(errStream io.Reader, errOutput *string, log func(...inter
 		if err != nil {
 			// we're ignoring ErrClosedPipe, as this is information
 			// the docker container ended streaming logs
-			if err != io.ErrClosedPipe && err != io.EOF {
+			if glog.V(2) && err != io.ErrClosedPipe && err != io.EOF {
```

Comment:
Ah... this is the code that we have in multiple places... here we had io.EOF, but not the glog.V(2)... perhaps you're just trying to make them look the same?